### ISSUE TITLE
--clu-level not working

### ISSUE THREAD

**Issue by spikeliu** [mapequation/infomap#153]
It seems like no matter what --clu-level I set, it will only print the top module to the .clu output. I tested it both with binary executable and Python API.

**Comment by spikeliu** [comment]
Also, when I use Python API "get_module" with "depth_level=-1", it doesn't return the bottom level, as I compare it with the corresponding part using "get_multilevel_modules"

**Comment by antoneri** [comment]
Hi, and thanks for reaching out!

I have confirmed your bug report in Infomap 1.2.0 and we will fix it as soon as possible!

However, in the python API, I get the correct results using the following:
```python
from infomap import Infomap
im = Infomap("-N5")
im.read_file("ninetriangles.net")
im.run()
im.write_clu("ninetriangles.clu", depth_level=-1) # output leaf modules
```

**Comment by antoneri** [comment]
Fixed in b82a0b1

We will add this in the next bug fix release. For now, you can clone the git repo or use the Python API as above.

Hope this helps!

**Comment by antoneri** [comment]
This has been fixed in Infomap 1.2.1, thank you for your patience!
